docs/docs/.vuepress/config.js

```diff
@@ -52,35 +70,52 @@ module.exports = {
         ],
       },
       {
-        title: "Workflows",
+        title: "Code",
         collapsable: false,
+        children: [
+          "/code/",
+          {
+            title: "Node.js",
+            children: ["/code/nodejs/"],
```

Comment:
I believe you'll need to add the docs you `git mv`d under this `children` section (e.g. workflow state, making HTTP requests, etc.)